### PR TITLE
feat(spanner): add disclaimer comment about gfe_latency metric

### PR DESCRIPTION
**Pull request by KiranmayiB**
Adding a disclaimer to the Cloud Spanner GFELatency sample as the java implementation uses the new API.
Disclaimer reads that the work is in progress for adding a similar API in Go.

### CODE REVIEW

**Comment by KiranmayiB** [comment]
DO NOT merge before November 26th 2021. Waiting for an internal approval.

File: spanner/spanner_snippets/spanner/spanner_opencensus_capture_gfe_metric.go
Comment (hengfengli) [inline spanner/spanner_snippets/spanner/spanner_opencensus_capture_gfe_metric.go:20]: nits: add `.` at the end.

**Review by hengfengli — APPROVED** [review]

**Pull request merged**
